src/app-config.mock.ts

```diff
@@ -157,7 +157,7 @@ export class AppMockConfig implements AbstractAppConfig {
       return '';
   }
 
-  getRDCommonDataAPIUrl(): string {
+  getCaseLinkingReasonCommonDataAPIUrl(): string {
```

Comment:
would be better to stick with 'getRDCommonDataAPIUrl', benefit is this is generic url not a specific url, and if other APIs in RDCommonData is required we don't need to create new entity, i.e. caseFlags are also in the scope of CommonData.
https://tools.hmcts.net/confluence/display/RTRD/Common+Reference+Data+APIs